WalletWasabi/BitcoinCore/P2pNode.cs

```diff
@@ -25,23 +28,22 @@ public P2pNode(Network network, EndPoint endPoint, MempoolService mempoolService
 			Stop = new CancellationTokenSource();
```

Comment:
`private CancellationTokenSource Stop { get; set; }` should be `private CancellationTokenSource Stop { get; }` as setter is not needed.

Reply:
ACK